public/editor/scripts/project/project.js

```diff
@@ -1,4 +1,5 @@
 define(function(require) {
+  var $ = require("jquery");
```

Comment:
It looks like we do it the first way throughout the rest of the Thimble code: https://github.com/mozilla/thimble.mozilla.org/search?q=jquery&type=Code&utf8=%E2%9C%93